### PR TITLE
Enable auth buttons in menu

### PR DESCRIPTION
**Pull request by krzysztofowsiany**
## Description
Translate register form
Testing registration and login
Centering registration forma and add santa animation

## Where should the reviewer start?
Register.cshtml

## Motivation and context
Because register page was ugly and in not national language #62 

## How has this been tested?
Start on IIS

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
 
 <h2>@ViewData["Title"]</h2>
-
-<div class="row">
+<div class="row content-centered">
     <div class="col-md-4">
         <form asp-route-returnUrl="@ViewData["ReturnUrl"]" method="post">
-            <h4>Create a new account.</h4>
+            <h4>Stwórz nowe konto.</h4>
             <hr />
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml]: remove this line

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml [hunk 2]
```diff
@@ -1,36 +1,43 @@
...
 }
 
 <h2>@ViewData["Title"]</h2>
-
-<div class="row">
+<div class="row content-centered">
     <div class="col-md-4">
         <form asp-route-returnUrl="@ViewData["ReturnUrl"]" method="post">
-            <h4>Create a new account.</h4>
+            <h4>Stwórz nowe konto.</h4>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml]: there is no need to display it - remove it

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml [hunk 3]
```diff
@@ -1,36 +1,43 @@
...
                 <input asp-for="Password" class="form-control" />
                 <span asp-validation-for="Password" class="text-danger"></span>
             </div>
             <div class="form-group">
-                <label asp-for="ConfirmPassword"></label>
+                <label asp-for="ConfirmPassword">Potwierdź hasło</label>
                 <input asp-for="ConfirmPassword" class="form-control" />
                 <span asp-validation-for="ConfirmPassword" class="text-danger"></span>
             </div>
-            <button type="submit" class="btn btn-default">Register</button>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml]: move button to center - add _div_ with _content-centered_ class

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml (hunk 1 above)
Comment (krzysztofowsiany) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml]: Fixed

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml (hunk 2 above)
Comment (krzysztofowsiany) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml]: Fixed

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml (hunk 3 above)
Comment (krzysztofowsiany) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Account/Register.cshtml]: Fixed

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError PasswordMismatch() { return new IdentityError { Code = nameof(PasswordMismatch), Description = "Niepoprawne hasło." }; }
+		public override IdentityError InvalidToken() { return new IdentityError { Code = nameof(InvalidToken), Description = "Niepoprawny token." }; }
+		public override IdentityError LoginAlreadyAssociated() { return new IdentityError { Code = nameof(LoginAlreadyAssociated), Description = "Użytkownik o wskazanym loginie już istnieje." }; }
+		public override IdentityError InvalidUserName(string userName) { return new IdentityError { Code = nameof(InvalidUserName), Description = $"Nazwa użytkownika '{userName}' jest nieprawidłowa, może jedynie zawierać litery i cyfry." }; }
+		public override IdentityError InvalidEmail(string email) { return new IdentityError { Code = nameof(InvalidEmail), Description = $"Adres email '{email}' jest niepoprawny." }; }
+		public override IdentityError DuplicateUserName(string userName) { return new IdentityError { Code = nameof(DuplicateUserName), Description = $"Nazwa użytkonwnika '{userName}' jest już zajęta." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: typo "użytkownika"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError DuplicateEmail(string email) { return new IdentityError { Code = nameof(DuplicateEmail), Description = $"Adres email '{email}' został juz zarejestrowany." }; }
+		public override IdentityError InvalidRoleName(string role) { return new IdentityError { Code = nameof(InvalidRoleName), Description = $"Rola '{role}' jest nieprawidłowa." }; }
+		public override IdentityError DuplicateRoleName(string role) { return new IdentityError { Code = nameof(DuplicateRoleName), Description = $"Rola '{role}' została przydzielona." }; }
+		public override IdentityError UserAlreadyHasPassword() { return new IdentityError { Code = nameof(UserAlreadyHasPassword), Description = "Użytkownik ma już ustawione hasło." }; }
+		public override IdentityError UserLockoutNotEnabled() { return new IdentityError { Code = nameof(UserLockoutNotEnabled), Description = "Blokada nie jest załączona." }; }
+		public override IdentityError UserAlreadyInRole(string role) { return new IdentityError { Code = nameof(UserAlreadyInRole), Description = $"Użytkownik jest już przycielony do role '{role}'." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: typo "przydzielony" "roli"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError UserNotInRole(string role) { return new IdentityError { Code = nameof(UserNotInRole), Description = $"Użytkowni nie jest w roli '{role}'." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: typo "użytkownik"
maybe use "Użytkownik nie ma roli {role"}

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError PasswordTooShort(int length) { return new IdentityError { Code = nameof(PasswordTooShort), Description = $"Hasło musi zawierać na końcu {length} znaków." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: maybe use "Hasło musi zawierać min {length} znaków"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError PasswordRequiresNonAlphanumeric() { return new IdentityError { Code = nameof(PasswordRequiresNonAlphanumeric), Description = "Hasło musi posiadać na końcu znak z poza alfabeturic character." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: typo "spoza"
maybe use "Hasło nie może posiadać znaków alfanumerycznych"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError PasswordRequiresDigit() { return new IdentityError { Code = nameof(PasswordRequiresDigit), Description = "Hasło musi zawierać na końcu cyfrę ('0'-'9')." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: maybe use "Hasło musi zawierać cyfrę (0-9)"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError PasswordRequiresLower() { return new IdentityError { Code = nameof(PasswordRequiresLower), Description = "Hasło musi zawierać na końcu małą litere ('a'-'z')." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: maybe use "Hasło musi zawierać małą literę (a-z)"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError PasswordRequiresUpper() { return new IdentityError { Code = nameof(PasswordRequiresUpper), Description = "Hasło musi zawierać na końcu dużą litere ('A'-'Z')." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: maybe use "Hasło musi zawierać dużą literę (A-Z)"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+		public override IdentityError InvalidToken() { return new IdentityError { Code = nameof(InvalidToken), Description = "Niepoprawny token." }; }
+		public override IdentityError LoginAlreadyAssociated() { return new IdentityError { Code = nameof(LoginAlreadyAssociated), Description = "Użytkownik o wskazanym loginie już istnieje." }; }
+		public override IdentityError InvalidUserName(string userName) { return new IdentityError { Code = nameof(InvalidUserName), Description = $"Nazwa użytkownika '{userName}' jest nieprawidłowa, może jedynie zawierać litery i cyfry." }; }
+		public override IdentityError InvalidEmail(string email) { return new IdentityError { Code = nameof(InvalidEmail), Description = $"Adres email '{email}' jest niepoprawny." }; }
+		public override IdentityError DuplicateUserName(string userName) { return new IdentityError { Code = nameof(DuplicateUserName), Description = $"Nazwa użytkonwnika '{userName}' jest już zajęta." }; }
+		public override IdentityError DuplicateEmail(string email) { return new IdentityError { Code = nameof(DuplicateEmail), Description = $"Adres email '{email}' został juz zarejestrowany." }; }
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/CustomIdentityErrorDescriber.cs]: typo "już"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Startup.cs [hunk 13]
```diff
@@ -1,75 +1,79 @@
...
 {
-    public class Startup
-    {
-        private const string DockerEnvName = "Docker";
-        public Startup(IHostingEnvironment env)
-        {
-            var configurationBuilder = new ConfigurationBuilder()
-                .AddJsonFile($"appsettings.{env.EnvironmentName}.json")
-                .AddEnvironmentVariables();
+	public class Startup
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Startup.cs]: why there are so many empty changes? do you have CodeMaid with default settings?

**Pull request merged**